### DISCUSSION TITLE
Missing municipality/hamlet details on self-hosted server

### DISCUSSION THREAD

**Discussion by RandyDautzenberg** [osm-search/Nominatim#3104] [Q&A]
Hi all,

Since a few weeks we’ve running a self-hosted Nominatim server now. The server is running like a gem, but we noticed a little difference in some of our queries sometimes.

![237e5baebfbae996764853bdc2671debcee8f3eb](https://github.com/osm-search/Nominatim/assets/3886485/84d07ae2-5f42-4888-9745-69ff69eb075d)

When we request a reverse query with lat/lon data on our self-hosted server, we are receiving (almost) the same information as the [nominatim.openstreetmap.org](http://nominatim.openstreetmap.org/) API, but unfortunately the municipality and also the hamlet is sometimes missing on our self-hosted server.

Does anyone have an idea how to add these missing municipality/hamlet details to our output?

Many thanks in advance!

Kind regards,
Randy Dautzenberg

**Comment by mtmail** [comment]
nominatim.openstreetmap.org runs a newer unreleased version of Nominatim. At the same time the last full reindex (full import) of the nominatim.openstreetmap.org data has been months ago while yours is probably more recent. Both make it hard to directly compare. You would need to install the latest unstable/master release and check if the same issue happens.

**Comment by Firefishy** [comment]
Asked the same question in 3 different places:
* https://help.openstreetmap.org/questions/87428/missing-municipalityhamlet-details-on-self-hosted-server
* https://community.openstreetmap.org/t/missing-municipality-hamlet-details-on-self-hosted-server/100798
* https://github.com/osm-search/Nominatim/discussions/3104

**Reply by mtmail** [reply]
Happy to answer any follow-up questions here.
